app/api/helpers/tasks.py

```diff
@@ -120,6 +121,19 @@ def resize_user_images_task(self, user_id, original_image_url):
         logging.exception('Error encountered while generating resized images for user with id: {}'.format(user_id))
 
 
+@celery.task(base=RequestContextTask, name='sponsor.logo.urls', bind=True)
+def sponsor_logos_url_task(self, event_id):
+    sponsor = Sponsor.query.filter_by(event_id=event_id).first()
```

Comment:
This will just do it for the first sponsor?

Reply:
Not really, since the function is called for each sponsor, every sponsor is taken care of @CosmicCoder96

Reply:
Even for each sponsor, the same sponsor is being fetched everytime

Reply:
Oh, I see the point now, will apply necessary changes and push the PR ASAP @iamareebjamal @CosmicCoder96